web/docs/tql2/language/modules.md

```diff
@@ -0,0 +1,12 @@
+# Modules
+
+TQL features namespacing in the form of *modules* to group related
+[operators](../operators.md) and [functions](../functions.md).
+
+A *module* separates the contained operation by two colons. For example, the
+[`package::add`](../operators/package/add.md) and
+[`package::remove`](../operators/package/remove.md) operators are in the
+`package` module.
+
+Modules are not hierarchical and have exactly one level of nesting. That is,
+the language does not support constructs of the form `a::b::c`.
```

Comment:
I think we can just leave this information out. It makes no practical difference to users.

Reply:
Hm, it was literally the first question I asked when I wanted to learn more about modules. To be fair, I'm not the ICP, but does it do harm?

Reply:
I think it's useless information for users that are not developers and that docs should be as concise as possible.

Reply:
Okay, removed in e492eb2.

Reply:
FWIW, the language _does_ support multiple levels of nesting (not tested, but that would be a bug otherwise).